packages/suite-desktop-core/e2e/support/pageActions/dashboardActions.ts

```diff
@@ -5,8 +5,16 @@ import { NetworkSymbol } from '@suite-common/wallet-config';
 import { waitForDataTestSelector } from '../common';
 
 class DashboardActions {
+    optionallyDismissFwHashCheckError(window: Page) {
+        // dismiss the error modal only if it appears (handle it async in parallel, not necessary to block the rest of the flow)
+        window
+            .$('[data-testid="@device-compromised/back-button"]')
```

Comment:
`window.getByTestId` throws if not found, while `window.$` just returns `null`

Reply:
Is this proper handling? I am no JS expert, so I don't know, but it looks, that you may sometimes call .then() on a null.

Reply:
that's why I added `?.click()` :slightly_smiling_face: 
It will either call it if truthy, or ignore if null

Reply:
Yeah, that's not what I meant, I thought that the .$ would return null, but it returns a promise whose value can be null. Everything okay! :)